.github/workflows/CODEOWNERS

```diff
@@ -0,0 +1,2 @@
+/.github/settings.yml @mmlb @micahhausler
+/.github/CODEOWNERS @mmlb @micahhausler
```

Comment:
This file is `/.github/workflows/CODEOWNERS`, should it be `/.github/CODEOWNERS`

Reply:
ah good catch